core/src/main/java/org/mskcc/oncotree/utils/CacheUtil.java

```diff
@@ -34,9 +40,17 @@ public static void resetTumorTypesByVersion(Version version) {
     }
 
     public static Map<String, TumorType> getTumorTypesByVersion(Version version) throws InvalidOncoTreeDataException {
+        logger.debug("getTumorTypesByVersion() -- looking for version '" + version.getVersion() + "' in cache");
```

Comment:
I never used logger before. But seems like this sentence is only needed when debug mode is enabled as well?

Reply:
@zhx828 That line won't execute when in debug mode (because it is logger.debug) but I wrapped the for loop in the check for debug mode because otherwise the loop would execute even in non-debug mode, even if nothing got printed.  It looks weird, but I think it is OK.